src/services/messages.js

```diff
@@ -15,10 +15,18 @@ export async function sendMessage(email, status = 0) {
 }
 
 export function subscribe(onMessage = (_message) => {}) {
+  const res = supabase
+    .from('messages')
+    .on('*', (payload) => {
+      console.log('Change received!', payload);
+    })
```

Comment:
On line 21, we will want to call our onMessage function that takes a parameter of the new attribute on the payload object. We can also omit our return statement as it is not needed.